services/app/src/components/avatarPicture/GhostAvatar.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import { Avatar } from 'antd';
+import { getAvatarPxSize, getAvatarFontSize } from '../../helpers/getAvatarSizes';
+
+const GhostAvatar = ({ avatarText, size }) => (
```

Comment:
Ghost naming is used to describe button styles in antd, please think of another name for this kind of avatar